site/source/docs/porting/Audio.rst

```diff
@@ -12,6 +12,7 @@ There are some implementation-specific aspects which are worth considering and a
 
 .. warning:: There used to be no additional flag to pass to the compiler in order to use OpenAL. However, specifying ``-lopenal`` as mentioned above should be considered mandatory (at some point in the future, it **will** be)!
 
+.. warning:: As Emscripten ships with its own implementation you must not link to any other implementation!
```

Comment:
How about "If you want use your own implementation of OpenAL and avoid linking in the version that ships with emscripten we recommend linking with `-sAUTO_JS_LIBRARIES=0`"

Reply:
@sbc100 It should be `-sAUTO_NATIVE_LIBRARIES=0` https://github.com/emscripten-core/emscripten/blob/8915f2b706ef0dd805150e6fb6d06bf8f0ad2446/src/settings.js#L1799C5-L1799C5

Reply:
I don't think so.     The openal implementation that we ship with emscripten is written in JS

Reply:
Then `Like AUTO_JS_LIBRARIES but for the native libraries such as libgl, libal and libhtml5.` in `settings.js` is wrong.

Reply:
Ah yes, you are correct openal has both a native and js component.